src/cgi/wwsympa.fcgi.in

```diff
@@ -7313,6 +7314,79 @@ sub do_auth_del {
 
     return ($in{'previous_action'} || 'sigindex');
 }
+# Deletes user from lists (requested by listmaster)
+sub do_mass_del {
+    wwslog('info', '(%s) (%s)', $in{'email'},
+        join(', ', split /\0/, $in{'lists'}));
+
+    # Access control.
+    return undef
+        unless (Sympa::is_listmaster($robot, $param->{'user'}{'email'}));
```

Comment:
`%required_privileges` would be better to restrict accesses by roles.

Reply:
The mass_del is only available from `/serveradmin/users` + search user, isn’t that function dedicated to listmaster only?

Reply:
By nature of HTTP, clients (users) may send /mass_del requests without going through those pages.

(Btw I thought that restriction by %required_privileges will issue more meaningful error message than returning undef.)

Reply:
Now using %required_privileges (https://github.com/sympa-community/sympa/pull/605/commits/c4b88f3c602e90d694360ad19813d39b93da18d3)

Reply:
Thanks.